apps/api-gateway/src/issuance/issuance.controller.ts

```diff
@@ -723,13 +723,15 @@ issueCredentialDto.type = 'Issuance';
         statusCode: HttpStatus.CREATED,
         message: ResponseMessages.issuance.success.create,
         data: getCredentialDetails
-      };    
-      const  webhookUrl = await this.issueCredentialService._getWebhookUrl(issueCredentialDto.contextCorrelationId).catch(error => {
+      };   
+      
+      const  webhookUrl = await this.issueCredentialService._getWebhookUrl(issueCredentialDto.contextCorrelationId, id).catch(error => {
         this.logger.debug(`error in getting webhook url ::: ${JSON.stringify(error)}`);
       });      
       if (webhookUrl) {
-        
-          await this.issueCredentialService._postWebhookResponse(webhookUrl, {data:issueCredentialDto}).catch(error => {
+        const plainIssuanceDto = JSON.parse(JSON.stringify(issueCredentialDto));
```

Comment:
Any reason to parsing data here?

Reply:
The data is being parsed to make a copy, so the original issueCredentialDto stays unchanged when sending it to the webhook.